Traceback (most recent call last):
  File "/home/ethany/drug-nlp/drug-NLP/mol_opt/run_drug_nlp.py", line 212, in <module>
    main()
  File "/home/ethany/drug-nlp/drug-NLP/mol_opt/run_drug_nlp.py", line 135, in main
    df = pd.read_csv(args.csv)
  File "/home/ethany/drug-nlp/drug-NLP/nlp/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 948, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/ethany/drug-nlp/drug-NLP/nlp/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 611, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/ethany/drug-nlp/drug-NLP/nlp/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1448, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/ethany/drug-nlp/drug-NLP/nlp/lib/python3.9/site-packages/pandas/io/parsers/readers.py", line 1705, in _make_engine
    self.handles = get_handle(
  File "/home/ethany/drug-nlp/drug-NLP/nlp/lib/python3.9/site-packages/pandas/io/common.py", line 863, in get_handle
    handle = open(
IsADirectoryError: [Errno 21] Is a directory: 'final_finetuning/results/'
